.codeclimate.yml

```diff
@@ -30,7 +30,7 @@ plugins:
     channel: eslint-7
   rubocop:
     enabled: true
-    channel: rubocop-0-88
+    channel: rubocop-0-92
```

Comment:
The latest version currently available is 0-92

https://github.com/codeclimate/codeclimate-rubocop/branches/all?utf8=%E2%9C%93&query=channel%2Frubocop